client/store.js

```diff
@@ -988,13 +988,14 @@ export let Actions = {
         });
     },
 
-    UpdateAccess(account, password) {
+    UpdateAccess(account, password, customFields) {
         assert(account instanceof Account, 'first param must be an account');
         assert(typeof password === 'string', 'second param must be the password');
```

Comment:
Can you please add a third assert: if customFields is not undefined, ensure it is an object and it has the two fields "name" and "value".